readthedocs/settings/dev.py

```diff
@@ -55,6 +56,8 @@ def DATABASES(self):  # noqa
     def LOGGING(self):  # noqa - avoid pep8 N802
         logging = super(CommunityDevSettings, self).LOGGING
         logging['formatters']['default']['format'] = '[%(asctime)s] ' + self.LOG_FORMAT
+        logging['handlers']['console']['level'] = 'DEBUG'
+        logging['loggers']['']['handlers'] = []
```

Comment:
Does this remove the double logging on development?

Reply:
Yea.